assets/app/view/game/map_controls.rb

```diff
@@ -81,13 +87,15 @@ def route_controls
           operator = all_operators.find { |o| o.name == operator_name }
           if operator
             store(:historical_routes, generate_last_route(operator))
+            store(:historical_laid_hexes, last_laid_hexes(operator))
           else
             store(:historical_routes, [])
+            store(:historical_laid_hexes, nil)
           end
         end
 
         @route_input = render_select(id: :route, on: { input: route_change }, children: operators)
-        h('label.inline-block', ['Show Last Route For:', @route_input])
+        h('label.inline-block', ['Show Last Route and Track For:', @route_input])
```

Comment:
I’d prefer "Tile", because tiles consist of more than tracks. (The same for comments in other files.)

Reply:
Changed Track to Tile